official-antistasi-community/A3-Antistasi
# Fix Townnames in Tasks and City Info.

## What type of PR is this.
1. [x] Bug
2. [x] Change
3. [ ] Enhancement

### What have you changed and why?
Information:
As Initzones uses the Displaynames for CitiesX we don't need to "Translate" them with Location.sqf anymore, we can use them directly.
Switched all Calls for Location.sqf to use the variable thats being used.
removed one faulty PVP Line in AS Official.
Changed on Line in CityInfo so it will display the "Description" Name instead of Worldname, as Worldname is sometimes a weird looking name.

### Please specify which Issue this PR Resolves.
closes #2295

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
Load any Map like Tanoa, Cherno, Malden, use Map Info (City Info), all Townnames should be easily reconiseable.
********************************************************
Notes:
